jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c

```diff
@@ -297,7 +273,7 @@ ecma_builtin_object_prototype_dispatch_routine (uint16_t builtin_routine_id, /**
 #else /* !ENABLED (JERRY_ESNEXT) */
       ecma_value_t this_parameter = ecma_make_object_value (obj_p);
 #endif /* ENABLED (JERRY_ESNEXT) */
-      ret_value = ecma_builtin_object_prototype_object_to_locale_string (this_parameter, obj_p);
+      ret_value = ecma_builtin_object_prototype_object_to_locale_string (this_parameter);
```

Comment:
This way the ToObject wil be called twice on the `this_parameter` argument: One is a bit above where the `obj_p` is initialized and the other one is in the underlying invoke method.